.github/workflows/autotest.yml

```diff
@@ -1,12 +1,14 @@
 name: Run all the autotests
 
 on:
-  - pull_request
-  - push
+  pull_request:
+  push:
+  schedule:
+    - cron: '0 3 * * *'
```

Comment:
Monday till Friday should be enough (or shifted by one if you want to rather run the tests on Saturday morning then Monday morning).
`0 3 * * 1-5`

Reply:
IMO, nightly/daily builds should happen every day regardless of weekends.

Reply:
I would also do it every day (for now). We are doing already a big reduction of tests with removing a large part of the jobs for the PR tests. So even for environmental considerations, this is a net positive step. If we see that it is really not needed, we can still remove.